bin/crew

```diff
@@ -471,6 +471,22 @@ def help(pkgName)
   end
 end
 
+def cache_build
+  @build_cachefile = "#{CREW_CACHE_DIR}#{@pkg.name}-#{@pkg.version}-build-#{@device[:architecture]}.tar.zst"
+  if CREW_CACHE_ENABLED && File.writable?(CREW_CACHE_DIR)
+    puts 'Caching build dir...'
+    Dir.chdir "#{CREW_BREW_DIR}#{@extract_dir}" do
+      # Do not use --exclude-vcs to exclude .git
+      # because some builds will use that information.
+      system "tar c#{@verbose} \
+      $(find -mindepth 1 -maxdepth 1 -printf '%P\n') \
```

Comment:
Include the whole `#{@extract_dir}` with '.'?

Reply:
I'm just using one dir name now...